tests/lib/rules/file-extension-in-import.js

```diff
@@ -313,21 +313,28 @@ new RuleTester({
             options: ["never", { ".json": "always" }],
             errors: [{ messageId: "forbidExt", data: { ext: ".mjs" } }],
         },
+
         {
+            // name: '.js has a higher priority than .json'
             filename: fixture("test.js"),
             code: "import './multi'",
-            output: null,
+            output: "import './multi.js'",
             options: ["always"],
-            errors: [
-                { messageId: "requireExt", data: { ext: ".cjs or .mjs" } },
-            ],
+            errors: [{ messageId: "requireExt", data: { ext: ".js" } }],
+        },
+        {
+            filename: fixture("test.js"),
+            code: "import './multi.js'",
+            output: null,
+            options: ["never"],
+            errors: [{ messageId: "forbidExt", data: { ext: ".js" } }],
         },
         {
             filename: fixture("test.js"),
-            code: "import './multi.cjs'",
+            code: "import './multi.json'",
             output: null,
             options: ["never"],
-            errors: [{ messageId: "forbidExt", data: { ext: ".cjs" } }],
+            errors: [{ messageId: "forbidExt", data: { ext: ".json" } }],
         },
```

Comment:
These multi tests did not make sense, as you cant import `.cjs` or `.mjs` files without a file extension. (that I know of / found)

I have thus swapped the test to use `.js` and `.json`.
This means that we can use the correct import priority as a fixer.

Reply:
:thinking: We can try to do that!